### PR TITLE
colibri2 and secure octo 

### PR DESCRIPTION
**Pull request by JonathanLennox**

### CODE REVIEW

File: src/main/resources/reference.conf
Comment (bgrozev) [inline src/main/resources/reference.conf:158]: Note to self: flip this before merging.

File: src/main/kotlin/org/jitsi/jicofo/conference/colibri/v2/ColibriV2SessionManager.kt
Comment (JonathanLennox) [inline src/main/kotlin/org/jitsi/jicofo/conference/colibri/v2/ColibriV2SessionManager.kt]: Is this a "should not happen", a "could happen in a race", or normal?

Comment (bgrozev) [reply src/main/kotlin/org/jitsi/jicofo/conference/colibri/v2/ColibriV2SessionManager.kt]: It's actually a "happens during normal operation because of awkward APIs": bridge goes down, JitsiMeetConference calls colibriSessionManager.removeBridges (which in v2 removes the participants), JitsiMeetConfernce calls removeParticipants. I've been thinking about how to avoid it, I think I'll just hide the message for now. When we remove colibri v1 we can clean it up.

**Review by bgrozev — APPROVED** [review]
Approving with @JonathanLennox 's consent

**Review by JonathanLennox — COMMENTED** [review]
I approve, but since this is nominally my PR, I can't stamp it.

**Pull request merged**
